.github/workflows/update-terraform-providers.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
             Check that all providers build with:
             ```
-            @ofborg build terraform-full
+            @ofborg build terraform.full
```

Comment:
How about `@ofborg build terraform-providers`, would that not work?

Reply:
No, it does work locally but ofborg doesn't like it. I think that's why I was using `terraform-full` originally.